testdata/project-v3-addon-and-grafana/Makefile

```diff
@@ -68,14 +68,40 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+# The build is supported with BuildKit enabled
+# If you disable this option then the manager image will be built using linux/amd64 by default
```

Comment:
Shouldn't it be local architecture by default? And is it safe to assume that all builders will be running from linux?

Reply:
@jaypoulz 

> Shouldn't it be local architecture by default?

This target is used so we are able to test the solutions out. So, it needs to be compatible by default with the local env. Authors can create releases and build the images according to their needs. Usually, they will use Goreleser or something like that probably. Anyway, if they want to use the target in a CI to do the releases see that it has the info that they can use the platform. 

> And is it safe to assume that all builders will be running from linux?
Yes, because the tool only supports Linux and Mac
Therefore, for mac platform it will be always linux.
Example: darwin/arm64 will have the docker BUILDPLATFORM linux/arm64

Reply:
It is outdated now, so i am closing this one